pom.xml

```diff
@@ -1117,6 +1165,54 @@
             </executions>
           </plugin>
           <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <version>3.0.0</version>
+              <executions>
+                  <!-- TODO as above -->
+                  <execution>
+                      <id>download node for Linux</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:node:${node.version}:tar.gz:linux-x64</artifact>
+                      </configuration>
+                  </execution>
+                  <execution>
+                      <id>download node for Darwin</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:node:${node.version}:tar.gz:darwin-x64</artifact>
+                      </configuration>
+                  </execution>
+                  <execution>
+                      <id>download node for Windows</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:node:${node.version}:exe:windows-x64</artifact>
+                      </configuration>
+                  </execution>
+                  <execution>
+                      <id>download yarn</id>
+                      <phase>initialize</phase>
+                      <goals>
+                          <goal>get</goal>
+                      </goals>
+                      <configuration>
+                          <artifact>com.github.eirslett:yarn:${yarn.version}:tar.gz</artifact>
```

Comment:
Actually this does not work yet; the plugin downloads this artifact to a different, and incorrect, location. Filed https://github.com/eirslett/frontend-maven-plugin/pull/605 to fix that; once merged & released, can correct the `frontend-version` here.